### ISSUE TITLE
katana issue now can't get any endpoint or parameter on a list of domains before is good but now I don't know what happened

### ISSUE THREAD

**Issue by Koraprognan** [projectdiscovery/katana#995]
this is command I use before and now "katana -u final_live.txt -d 5 -ps -pss waybackarchive,commoncrawl,alienvault -f qurl -jc -xhr -kf -fx -fs dn -ef woff,css,png,svg,jpg,woff2,jpeg,gif,svg -o urls.txt" 

But I wonder now that can't get any endpoint or parameter on my lists

**Comment by Koraprognan** [comment]
Hey bro I'll update my kali repository and now quite good hope don't have encounter any error in the future

**Comment by LiuHao00300** [comment]
You got any solution yet bro??

**Comment by gulfam-ali-j** [comment]
still its shwoing like this error for passive enumeuration  :-  " flag provided but not defined: -ps "

**Comment by dogancanbakir** [comment]
Yes, it is now an independent tool. You can check it out at https://github.com/projectdiscovery/urlfinder.
